text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,65 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
-
+    __________________________________________________________________
+    Hello, Notnow here!
+    What can I do for you?
+    __________________________________________________________________
+    __________________________________________________________________
+    added: [D][ ] do homework (by: no idea :-p)
+    __________________________________________________________________
+    __________________________________________________________________
+    added: [D][ ] return book (by: Sunday)
+    __________________________________________________________________
+    __________________________________________________________________
+    added: [E][ ] project meeting (from: Mon 2pm to: 4pm)
+    __________________________________________________________________
+    __________________________________________________________________
+    added: [T][ ] borrow book
+    __________________________________________________________________
+    __________________________________________________________________
+    1.[D][ ] do homework (by: no idea :-p)
```

Comment:
Consider adding a space between the number and the item to make it easier to read, i.e. `1. Something` instead of `1.Something`.